testing/environments/prev-minor.yml

```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.15.1
```

Comment:
This is called prev-minor so it should potentially stay on 7.15.1?

Reply:
Yep, I have changed it to 7.15.2, let's see

Reply:
well, it seems it fixed the related tests